docs/design/automatic-updates

```diff
@@ -0,0 +1,37 @@
+
+#Storage Node automatic updates DesignDocument
+
+##OVERVIEW
```

Comment:
Use proper casing, the uppercase is for styling not for input. This messes up screen-readers.